### PR TITLE
added remove, back command on music system

### PR DESCRIPTION
**Pull request by LackyPal**

### CODE REVIEW

**Comment by LackyPal** [comment]
Sorry,wait some time. Need to add translation

File: src/commands/music/remove.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      if (isNaN(Number(songNo))) {
+        return message.channel.send(lang.LEVELS.PROVIDE_VALID_NR);
+      }
+  
+      if (Number(songNo) < 1) {
+        return message.channel.send(lang.MUSIC.BETWEEN_1_TOTALQUEUE);
+      }
+  
+      if (Number(songNo) > queue.tracks.length) {
+        return message.channel.send(lang.MUSIC.BETWEEN_1_TOTALQUEUE);
```
Comment (casperiv0) [inline src/commands/music/remove.ts]: `BETWEEN_1_TOTALQUEUE` does not exist in the translation file, can you please add it.

File: src/commands/music/remove.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import Command from "../../structures/Command";
+import Bot from "../../structures/Bot";
+
+export default class RemoveCommand extends Command {
+  constructor(bot: Bot) {
+    super(bot, {
+      name: "remove",
+      description: "Remove a song from the queue",
+      category: "music",
+      aliases: [],
```
Comment (casperiv0) [inline src/commands/music/remove.ts:11]: Please add `requiredArgs: ["track-number"]` option

File: src/commands/music/remove.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        return message.channel.send(lang.MUSIC.BETWEEN_1_TOTALQUEUE);
+      }
+  
+      if (Number(songNo) > queue.tracks.length) {
+        return message.channel.send(lang.MUSIC.BETWEEN_1_TOTALQUEUE);
+      }
+  
+      if (!songNo) {
+        return message.channel.send(lang.LEVELS.PROVIDE_VALID_NR);
+      }
```
Comment (casperiv0) [inline src/commands/music/remove.ts]: This can be removed since we're using `requiredArgs`

**Review by casperiv0 — CHANGES_REQUESTED** [review]
Please check the comments :). You can remove the `search` command since we're not going to using that.

**Comment by LackyPal** [comment]
I think I've done. You can check and merge. If any Error, please inform me

**Review by casperiv0 — APPROVED** [review]
Looks good!

**Comment by casperiv0** [comment]
I will merge, and quickly test it 🎉

**Comment by casperiv0** [comment]
@L0SER8228 it seems to not work lol: 
image

Feel free to re-open a PR with a fix :)

**Pull request merged**